From 83608dbd659c0f48b2b5408d79f3ff61adcb6e7b Mon Sep 17 00:00:00 2001
From: Danct12 <danct12@disroot.org>
Date: Sun, 23 Jun 2024 06:42:53 +0000
Subject: [PATCH 1/7] arm64: dts: rockchip: pinetab2: Always enable VDD_GPU_NPU
 regulator

It turns out you aren't supposed to turn off the regulator or the
a panic will occur when the driver tries to turn it back on.
---
 arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
index db40281eafbe..a9932ffeef92 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
@@ -399,6 +399,8 @@ regulator-state-mem {
 			};
 
 			vdd_gpu_npu: DCDC_REG2 {
+				regulator-always-on;
+				regulator-boot-on;
 				regulator-min-microvolt = <500000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
-- 
2.45.2

